src/platform/packages/shared/kbn-alerts-as-data-utils/src/schemas/create_schema_from_field_map.ts

```diff
@@ -347,6 +347,13 @@ const getSchemaFileContents = (lineWriters: Record<string, LineWriter>, schemaPr
 const writeGeneratedFile = (fileName: string, contents: string) => {
   const genFileName = path.join(PLUGIN_DIR, fileName);
   try {
+    // Check if file exists and content is the same
```

Comment:
I added this because the test for generating the alert schema was consistently failing on my local setup.
It was always writing the schema file, even when no changes had been made, which caused Kibana to restart each time and become unavailable.
Am I missing something or doing something wrong here?

<img width="658" height="362" alt="Screenshot 2025-09-04 at 10 16 02" src="https://github.com/user-attachments/assets/4c3f1186-9ce7-4ea4-a834-1a70a46e3b2e" />